### PR TITLE
Add support for Grafana & Prometheus monitoring

### PR DESCRIPTION
**Pull request by nathanklick**
## Description

Adds support for automatic deployment of Prometheus and Grafana via the docker compose definitions. Additionally, adds automatic provisioning of a basic set of dashboards. 

Credit for this change goes to @beeradb as all I did was the final polish work.

### CODE REVIEW

**Review by georgi-l95 — CHANGES_REQUESTED** [review]
Super nice addition. Just need to fix the test, you need to add some additional lines and we're good to go. 
Because we are using `mustache` to template and use the node with different configurations (testnet/mainnet/previewnet/latest), we need to add those lines in the templates as well.

**Review by georgi-l95 — APPROVED** [review]
nvm, i've added them.

File: compose-network/prometheus/prometheus.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+global:
+  scrape_interval: 15s
+  evaluation_interval: 15s
+
+scrape_configs:
```
Comment (hendrikebbers) [inline compose-network/prometheus/prometheus.yml:5]: 3 s is the snapshot interval of the platform. We should confuter that here, too

**Review by hendrikebbers — COMMENTED** [review]
A good solution for local node. But now we need a workflow how dashboards that are created somewhere will end here.

File: compose-network/prometheus/prometheus.yml (hunk 1 above)
Comment (beeradb) [reply compose-network/prometheus/prometheus.yml:4]: Done. I also removed the `evaluation_interval`, as we're not evaluating rules in this config, it doesn't feel necessary to modify the default.

**Pull request merged**